src/components/Details/TranscriptItem/TranscriptItem.tsx

```diff
@@ -109,58 +125,60 @@ const TranscriptItem: FC<TranscriptItemProps> = ({
   );
 
   return (
-    <Item>
-      <Text>
-        <Highlighter
-          searchWords={(searchQuery?.trim() || "").split(/\s+/g)}
-          autoEscape={true}
-          textToHighlight={text}
-        />
-      </Text>
-      <Container hasMultipleActions={handleTranscriptClick !== undefined}>
-        <Speaker>
-          {avatar}
-          <div>
-            {name}
-            <p>{startTime}</p>
-          </div>
-        </Speaker>
-        <div>
-          <Popup
-            position="top right"
-            content="Jump to sentence in video"
-            size="mini"
-            trigger={
-              <Button
-                aria-label="Jump to sentence in video"
-                className="mzp-c-button mzp-t-neutral"
-                onClick={handleVideoClick}
-              >
-                <PlayIcon />
-              </Button>
-            }
+    <div ref={transcriptItemRef}>
+      <Item>
+        <Text>
+          <Highlighter
+            searchWords={(searchQuery?.trim() || "").split(/\s+/g)}
+            autoEscape={true}
+            textToHighlight={text}
           />
-        </div>
-        {handleTranscriptClick && (
+        </Text>
+        <Container hasMultipleActions={handleTranscriptClick !== undefined}>
+          <Speaker>
+            {avatar}
+            <div>
+              {name}
+              <p>{startTime}</p>
+            </div>
+          </Speaker>
           <div>
             <Popup
               position="top right"
-              content="Jump to sentence in transcript"
+              content="Jump to sentence in video"
               size="mini"
               trigger={
                 <Button
-                  aria-label="Jump to sentence in transcript"
+                  aria-label="Jump to sentence in video"
```

Comment:
Were these just swapped? Hard to tell with the git diff.

Reply:
No, not swapped. Weird git diff